### PR TITLE
fix(editor): Add striped background to readonly canvas (no-changelog)

### PR DESCRIPTION
**Pull request by r00gm**
## Summary

Add striped background to canvas readonly mode
image
image

## Related Linear tickets, Github issues, and Community forum posts

[N8N-7782](https://linear.app/n8n/issue/N8N-7782/canvas-background-on-executions-tab-is-same-as-in-the-editor-tab-not)

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/editor-ui/src/components/canvas/Canvas.vue [hunk 1]
```diff
@@ -562,7 +564,12 @@ provide(CanvasKey, {
 
 		<CanvasArrowHeadMarker :id="arrowHeadMarkerId" />
 
-		<Background data-test-id="canvas-background" pattern-color="#aaa" :gap="GRID_SIZE" />
+		<Background data-test-id="canvas-background" pattern-color="#aaa" :gap="GRID_SIZE">
+			<template v-if="readOnly" #pattern-container>
+				<CanvasBackgroundStripedPattern :x="viewport.x" :y="viewport.y" :zoom="viewport.zoom" />
+				<rect x="0" y="0" width="100%" height="100%" fill="url(#diagonalHatch)" />
```
Comment (alexgrozav) [inline packages/editor-ui/src/components/canvas/Canvas.vue:570]: Could we move the `<rect>` inside of `CanvasBackgroundStripedPattern`? We should be able to render multiple roots like a Fragment without problems.

**Review by alexgrozav — CHANGES_REQUESTED** [review]
Small question. I'll approve if the change doesn't work.

File: packages/editor-ui/src/components/canvas/Canvas.vue (hunk 1 above)
Comment (r00gm) [reply packages/editor-ui/src/components/canvas/Canvas.vue:570]: it should work but eslint complains
image

Comment (r00gm) [reply packages/editor-ui/src/components/canvas/Canvas.vue]: done

**Review by alexgrozav — APPROVED** [review]
Thanks for addressing the changes. 🚀

**Comment by janober** [comment]
Got released with [`n8n@1.65.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.65.0)

**Pull request merged**